api/deployments/models/scheduled_batch.go

```diff
@@ -1,31 +1,33 @@
 package models
 
+import "time"
+
 // ScheduledJobSummary holds general information about scheduled job
 // swagger:model ScheduledJobSummary
 type ScheduledJobSummary struct {
 	// Name of the scheduled job
 	//
-	// required: false
+	// required: true
 	// example: job-component-20181029135644-algpv-6hznh
 	Name string `json:"name"`
 
 	// Created timestamp
 	//
-	// required: false
-	// example: 2006-01-02T15:04:05Z
-	Created string `json:"created,omitempty"`
+	// required: true
+	// swagger:strfmt date-time
+	Created time.Time `json:"created"`
```

Comment:
Created is nilable for jobs. It is the time the k8is job was created.
Changing this to nilable will also affect other changes in the PR, so make sure you check where it is used.

Reply:
Fixed :)